docs/services/web_power_switch.rst

```diff
@@ -1,14 +1,48 @@
 Web Power Switch
 ================
 
+This service controls a power switch that is controllable over the internet.
+
+For some examples of commercially available web power switches:
+https://controlbyweb.com/webswitch/
+https://dlidirect.com/products/new-pro-switch
```

Comment:
Can you list the ones that have been run successfully with catkit2?

Reply:
Done